parser-typechecker/src/Unison/Hashing/V2/BuildHashable.hs

```diff
@@ -0,0 +1,140 @@
+module Unison.Hashing.V2.BuildHashable where
```

Comment:
IIRC we wanted to only export certain things for safety, not sure quite where this implementation ended up on that,
was there any plan to restrict exports here?